app.js

```diff
@@ -1,23 +1,45 @@
 'use strict';
 
-function CookieStore (name, minCustomers, maxCustomers, avgCookies, hourlyCookieCount) { // this is a constructor function taking in 4 parameters.
+function CookieStore (name, minCustomers, maxCustomers, avgCookies, hourlyCookieSales) { // this is a constructor function taking in 4 parameters.
   this.name = name || 'Unknown'; // adding || (or) inputs a value if it doesn't meet true.
   this.minCustomers = minCustomers;  // the context of this is going to change. The keyword 'this' knows what it represents by the surrounding context
   this.maxCustomers = maxCustomers; // argument on the right is going into the parameter in the function.
   this.avgCookies = avgCookies;
-  this.hourlyCookieCount = hourlyCookieCount || [];// assigning an empty array []
+  this.hourlyCookieSales = hourlyCookieSales || [];// assigning an empty array []
   this.totalCountForDay = 0;
-  this.businessHours = ['07am ', '08am ', '09am ', '010am ', '011am ', '12pm ', '01pm ', '02pm ', '03pm ', '04pm ', '05pm ','06pm ','07pm ', '08pm '];
+  this.businessHours = ['06am','07am ', '08am ', '09am ', '010am ', '011am ', '12pm ', '01pm ', '02pm ', '03pm ', '04pm ', '05pm ','06pm ','07pm ', '08pm ', 'Daily Location Total'];
 }
 
 CookieStore.prototype.getAvgCookieCount = function () { // Add method with a prototype
-  for (var hours = 0; hours < this.businessHours.length; hours++) {
+  for (var hours = 0; hours < this.businessHours.length - 1; hours++) {
     var avgCookiesPerhour = Math.floor(Math.random() * ((this.maxCustomers - this.minCustomers + 1) + this.minCustomers) * this.avgCookies);
-    this.hourlyCookieCount.push(avgCookiesPerhour);
+    this.hourlyCookieSales.push(avgCookiesPerhour);
     this.totalCountForDay += avgCookiesPerhour;
   }
 };
 
+CookieStore.prototype.renderRow = function () {
```

Comment:
Nice job on the method!  The the code is basically the same, but hopefully by the time you finish this lab you will notice how powerful having it in a prototype can be.